src/content/app/genome-browser/components/zmenu/Zmenu.test.tsx

```diff
@@ -50,6 +53,21 @@ jest.mock('./ZmenuInstantDownload', () => () => (
   <div>ZmenuInstantDownload</div>
 ));
 
+const chrName = faker.lorem.word();
+const startPosition = faker.datatype.number({ min: 1, max: 1000000 });
+const endPosition =
+  startPosition + faker.datatype.number({ min: 1000, max: 1000000 });
+
+const mockState = {
+  browser: {
+    browserGeneral: {
+      actualChrLocations: {
+        human: [chrName, startPosition, endPosition] as ChrLocation
+      }
+    }
+  }
+};
```

Comment:
Well, now you can write a proper test case for the logic that you added 😄 

- replace the mock redux store with the real redux store
- dispatch an event that changes the chromosome location of human
- confirm that the zmenu was destroyed by observing the consequences that you can control (`setZmenus` would be called on `useGenomeBrowser`)